Dockerfile

```diff
@@ -26,7 +26,18 @@ RUN mvn clean package -DskipTests=true -Dspotbugs.skip=true -Dpmd.skip=true
 
 # Runtime Container
 FROM amazoncorretto:17-alpine
+
+# Setup rights for overwriting frontend runtime variables
+COPY --from=VUE /frontend/setup-runtime.sh setup-runtime.sh
+
+RUN \
+    mkdir public && \
+    chmod a+x ./setup-runtime.sh && \
+    chmod -R a+rw ./public
+
+COPY --from=VUE /frontend/dist/env.js ./public/env.js
 COPY --from=MAVEN /home/maven/business-partner-agent/target/business-partner-agent*SNAPSHOT.jar business-partner-agent.jar
 
+
 EXPOSE 8080
-CMD java -XX:+UnlockExperimentalVMOptions -Dcom.sun.management.jmxremote ${JAVA_OPTS} -jar business-partner-agent.jar
+CMD ./setup-runtime.sh public/env.js business-partner-agent.jar && java -XX:+UnlockExperimentalVMOptions -Dcom.sun.management.jmxremote ${JAVA_OPTS} -jar business-partner-agent.jar
```

Comment:
How does this impact start performance?

Reply:
About 5 seconds, the script will start after everything else has been set up